app/assets/stylesheets/aleph.scss

```diff
@@ -0,0 +1,11 @@
+// Place all the styles related to the aleph controller here.
+// They will automatically be included in application.css.
+// You can use Sass (SCSS) here: http://sass-lang.com/
+
+.fa-check {
+  color: green;
```

Comment:
I'd like to use the color variables here: $success and $error (i know it's not an error but hey). This means we'll have to get this in the right area to know the variables exist. Let me know if you need help with that (or you can put this in bento.scss)